03-casestudies.Rmd

```diff
@@ -352,14 +352,17 @@ Interactive graph have its own specific advantages. It helps bridge the gap betw
 ### Malaria
 The authors of Vizwiz redesigned "The Seasonality of Confirmed Malaria Cases in Zambia Southern Province" by pointing out what works well, what could be improved, and why their new visualization will be better [@vizwiz].
 
-This chart below shows number of malaria cases reported for health facilities and community health workers and a comparison between the two over the years. From this chart we can clearly see that as summer approaches, cases of malaria increase indicating a seasonality. The colors are also distinct from each other. 
 Original Version: ![Original visualization of malaria cases](images/malaria_original.png)
+([Source]https://www.path.org/visualize-no-malaria/)
```

Comment:
should the link be within another parenthesis?

Reply:
I changed it. Can you see?

Reply:
I fixed again!